### ISSUE TITLE
Security Feedback

### ISSUE THREAD

**Issue by Lohrer** [micro-nova/AmpliPi#198]
- [ ] Put a basic firewall in via iptables.
  - [ ] Allow SSH, HTTP, development port web UI port, and both standard and development updater ports.
  - [ ] Figure out which ports required for streams and allow them.
- [x] Generate a unique password for "pi"

**Comment by dlasher** [comment]
+1

You have some talented people happy to help audit/eval - make use of them. ;)

**Comment by boulwarek** [comment]
Shouldn't the firewall rules also include Airplay and DLNA ports?

**Comment by Lohrer** [comment]
Yes, and there are probably more! We haven't checked what all is required yet. I see I forgot to list the updater ports too.
